package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "td-js-sdk",
-  "version": "1.6.0",
+  "version": "1.6.1",
```

Comment:
Reset this change as well. I'll bump the version when I make the release. Since we follow semver it has to be `v1.7.0`; it's not a bug fix.